2_drivers.yml

```diff
@@ -17,13 +17,39 @@ targets:
   #out_data/placeholder_for_gefs_archive:
    # command: copy_files()
     
-  #out_data/palceholder_for_nldas:
-   # command: copy_files()
-   
+  # NLDAS Files
+  ## NLDAS_time[0.377900]_x[397]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[397]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[397]_y[136].csv',
+    to = target_name)
+  ## NLDAS_time[0.377900]_x[401]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[401]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[401]_y[136].csv',
+    to = target_name)
+  
+  ## NLDAS to reservoir meteo rds  
+  log/reservoir_meteo_files.yml:
```

Comment:
Will this work if this target is built here? I ask because it relies on `out_data/reservoir_nml_list.rds` which is build from `out_data/NLDAS_time[0.377900]_x[397]_y[136].csv` and `out_data/NLDAS_time[0.377900]_x[401]_y[136].csv` and that happens in the remake.yml